services/src/functional-tests/src/main/groovy/org/openkilda/functionaltests/helpers/FlowHistoryAction.groovy

```diff
@@ -0,0 +1,24 @@
+package org.openkilda.functionaltests.helpers
+
+enum FlowHistoryAction {
+    //action
+    FLOW_CREATING("Flow creating"),
+    FLOW_UPDATING("Flow updating"),
+    FLOW_REROUTING("Flow rerouting"),
+
+    //nested action
+    SET_THE_FLOW_STATUS_TO_UP("Set the flow status to UP."),
+    SET_THE_FLOW_STATUS_TO_DOWN("Set the flow status to down."),
+    FLOW_WAS_VALIDATED_SUCCESSFULLY("Flow was validated successfully"),
```

Comment:
Ok for now. JFYI it is also possible to have `Recalculate flow status to DOWN.`, `Recalculate flow status to UP.`, `Recalculate flow status to In Progress.` during Flow Rerouting.